src/Wallabag/ImportBundle/Command/ImportCommand.php

```diff
@@ -98,9 +98,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new Exception(sprintf('File "%s" not found', $input->getArgument('filepath')));
         }
 
-        // Turning off doctrine default logs queries for saving memory
-        $this->entityManager->getConnection()->getConfiguration()->setSQLLogger(null);
```

Comment:
It's not available anymore?

Reply:
setSQLLogger is deprecated and suggested to be replaced by a middleware.
I went thr easy way by changing the configuration, see the change of config_test.yml in the same commit

Reply:
The idea wasn't to disabled it only in test env. We've got issue while using the import command in production.
Is it hard to implement a middleware to disable it only in that context?

Reply:
I'll simply remove the commits related to Doctrine deprecation notices, as they are not mandatory to the Symfony 5 upgrade.
Will deal with that in a future PR

Reply:
Code and description updated!

Reply:
> Is it hard to implement a middleware to disable it only in that context?

Here the PR implementing the logging middleware for reference: https://github.com/doctrine/dbal/pull/4967
I think the solution would be to remove it from the middleware stack instead, if possible